### PR TITLE
Fix a few items found with flake8. Added pgpdump to requirements.

### PR DESCRIPTION
**Pull request by wdoekes**

### CODE REVIEW

**Comment by SunDwarf** [comment]
You might wanna change that to pgpdump3 :)

**Comment by wdoekes** [comment]
Ah right. Thanks.

Fixed, along with a few other things. Also added the rst files so setup.py long_description generation works.

File: pgp/commands/gpg/io_helper.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def __init__(self, stdin, stdout, stderr, output, max_output, status_fd,
         self.enable_special_filenames = enable_special_filenames
 
         if not no_greeting:
-            print('Welcome!', file=self.stdout)
+            self.stdout.write('Welcome!\n')
```
Comment (mitchellrj) [inline pgp/commands/gpg/io_helper.py]: I'd prefer `from __future__ import print_function` as the fix for this.

File: pgp/pinentry/getpass.py [hunk 2]
```diff
@@ -44,15 +44,15 @@ def get_passphrase(self, prompt=None, error_message=None,
             raise KeyError(cache_id)
 
         if prompt:
-            print(prompt, file=stream)
+            stream.write(prompt + '\n')
```
Comment (mitchellrj) [inline pgp/pinentry/getpass.py]: as above, `from __future__ import print_function` would make this look more elegant and forward-thinking.

**Comment by wdoekes** [comment]
updated

**Pull request merged**